packages/legend-studio/src/components/editor/edit-panel/service-editor/ServiceTestEditor.tsx

```diff
@@ -683,13 +689,48 @@ export const ServiceTestEditor = observer(
                   </div>
                 </div>
                 <div className="panel__header__actions">
+                  <div
```

Comment:
This is a tiny feature people may not even need to know about them, why do we need to make it so apparent? Consider the mapping test/execution runner panel and this

Reply:
We don't know this is a tiny feature. There is no clear cut Icon to make the distinction. I would rather make this apparent.